apps/src/lab2/levelEditors/validations/EditCondition.tsx

```diff
@@ -78,45 +102,54 @@ const EditCondition: React.FunctionComponent<EditConditionProps> = ({
           );
         })}
       </select>
-      {hasValue && (
+      {hasValueType && (
         <>
-          <label htmlFor="conditionValue" className={moduleStyles.label}>
-            Value:
-          </label>
-          {useInput && (
-            <input
-              type={isNumeric ? 'number' : 'text'}
-              name="conditionValue"
-              id="conditionValue"
-              value={condition.value}
-              onChange={e => {
-                condition.value = isNumeric
-                  ? parseInt(e.target.value)
-                  : e.target.value;
-                onConditionChange(condition, index);
-              }}
-            />
-          )}
-          {useDropdown && (
-            <select
-              className={moduleStyles.conditionValueDropdown}
-              name="conditionValue"
-              id="conditionValue"
-              value={condition.value || dropdownOptions[0]}
-              onChange={e => {
-                condition.value = e.target.value;
-                onConditionChange(condition, index);
-              }}
-            >
-              {dropdownOptions.map((option, index) => {
-                return (
-                  <option key={index} value={option}>
-                    {option}
-                  </option>
-                );
-              })}
-            </select>
-          )}
+          <label className={moduleStyles.label}>Value:</label>
+          {valueTypes.map((type, i) => {
```

Comment:
Going by the screenshots, can we have a small gap between each of these?  That will allow things like a focus glow to be seen properly.